src/pytest_markdown_docs/plugin.py

```diff
@@ -294,8 +300,13 @@ def find_object_tests_recursive(
                     or "<Unnamed obj>"
                 )
                 fence_syntax = FenceSyntax(self.config.option.markdowndocs_syntax)
+                markdown_it_parsers = (
```

Comment:
Regarding the list comment above - maybe it's better to validate the hook return value here (making sure it's at most one specification, since that's what we support) and then just pass in the single MarkdownIt we want to use?

Reply:
With https://github.com/modal-labs/pytest-markdown-docs/commit/0be6a102d5662c2ccfcfd6abba7aaeb8f1edfce4 and https://github.com/modal-labs/pytest-markdown-docs/commit/5fe7f6fa06496a563485a5a1fba567e6a0fd31b2, this should  be a single `MarkdownIt` object.

The remaining user error is returning a non-MarkdownIt object:

```python
def pytest_markdown_docs_markdown_it():
    return "abc"
```

Do you think it's worth checking for this edge case?

Reply:
With the other changes I don't think we need to check for anything here 👍